scripts/publish-release

```diff
@@ -3,31 +3,29 @@
 set -e -o pipefail
 
 CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
-MAIN_BRANCH="main"
+RELEASE_PREFIX="release-"
 
-if [[ "${CURRENT_BRANCH}" != "${MAIN_BRANCH}" ]]; then
-  echo "scripts/publish-release must be run on the ${MAIN_BRANCH}."
-  echo "Please checkout the ${MAIN_BRANCH} branch with:"
-  echo "git checkout ${MAIN_BRANCH}"
+if [[ "${CURRENT_BRANCH}" != *"${RELEASE_PREFIX}"*  ]]; then
+  echo "scripts/publish-release must be run on a branch with this pattern: ${RELEASE_PREFIX}*."
   exit 1
 fi
 
-if !(command -v gh > /dev/null); then
-  echo "GitHub CLI is required for this release script."
-  echo "Please install GitHub CLI and try again:"
-  echo "https://github.com/cli/cli#installation"
-  exit 1
-fi
-
-# hostname is necessary just in case you are logged into the CLI
-# GitHub enterprise instance
-if !(gh auth status --hostname "github.com" > /dev/null 2>&1); then
-  echo "Not logged into github".
-  echo "Please run: gh auth login"
-  exit 1
-fi
-
-git pull --rebase origin "${MAIN_BRANCH}"
+#if !(command -v gh > /dev/null); then
+#  echo "GitHub CLI is required for this release script."
+#  echo "Please install GitHub CLI and try again:"
+#  echo "https://github.com/cli/cli#installation"
+#  exit 1
+#fi
+
+## hostname is necessary just in case you are logged into the CLI
+## GitHub enterprise instance
+#if !(gh auth status --hostname "github.com" > /dev/null 2>&1); then
+#  echo "Not logged into github".
+#  echo "Please run: gh auth login"
+#  exit 1
+#fi
```

Comment:
All commented-out code is related to GitHub releases and will be added once our service account is approved. It will be manually handled until then.